From 541b1f49b6b2c09b386e3f8292894e4fcecca6af Mon Sep 17 00:00:00 2001
From: hengluo <heng.luo@intel.com>
Date: Mon, 22 Oct 2018 13:47:23 +0800
Subject: [PATCH] Add update_engine to media_rw group

Allow update_engine to access the OTA file in removable storage.

Change-Id: I501485f174a0bf01ea36a9f8d22e0cfd218883f1
Tracked-On:
Signed-off-by: hengluo <heng.luo@intel.com>
---

diff --git a/update_engine.rc b/update_engine.rc
index a7d6235..90ca4c6 100644
--- a/update_engine.rc
+++ b/update_engine.rc
@@ -1,7 +1,7 @@
 service update_engine /system/bin/update_engine --logtostderr --logtofile --foreground
     class late_start
     user root
-    group root system wakelock inet cache
+    group root system wakelock inet cache media_rw
     writepid /dev/cpuset/system-background/tasks
     disabled
 
